linera-storage/src/lib.rs

```diff
@@ -177,7 +174,8 @@ pub trait Storage: Sized {
     /// Returns a map of the committees for the given epochs.
     async fn committees_for(
         &self,
-        mut epochs: BTreeSet<Epoch>,
+        min_epoch: Epoch,
+        max_epoch: Epoch,
```

Comment:
There's a degenerative case when we pass in (by mistake) a `min_epoch > max_epoch`. Maybe it's worth panicing in such case?

Reply:
I was wondering about this. Perhaps we could use `epoch_range: Range<Epoch>`.